prov/efa/src/efa_rma.c

```diff
@@ -129,6 +152,17 @@ ssize_t efa_rma_read(struct fid_ep *ep, void *buf, size_t len, void *desc,
 	return efa_rma_readv(ep, &iov, &desc, 1, src_addr, addr, key, context);
 }
 
+ssize_t efa_rma_read_ex(struct fid_ep *ep, void *buf, size_t len, void *desc,
+			fi_addr_t src_addr, bool self_comm,
+			uint64_t addr, uint64_t key,
+			void *context)
+{
+
+	efa_ep = container_of(ep, struct efa_ep, util_ep.ep_fid);
+	return efa_rma_post_read(efa_ep, &msg, 0, self_comm);
+}
+
+>>>>>>> 243baae05... use efa_rma_read_ex()
```

Comment:
A quick look, Is this a conflict we have not resolved?

Reply:
Rebase error. fixed in updated version.